pkg/kev/services_test.go

```diff
@@ -141,6 +143,189 @@ var _ = Describe("ServiceConfig", func() {
+
+						err := serviceConfig.validate()
+						Expect(err).To(Succeed())
+					})
+				})
+
+				Context("liveness", func() {
+					BeforeEach(func() {
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeType] = kubernetes.ProbeTypeHTTP.String()
+					})
+
+					It("fails when http probe has missing path", func() {
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeHTTPPort] = "8080"
+
+						err := serviceConfig.validate()
+						Expect(err).To(HaveOccurred())
+						Expect(err.Error()).To(ContainSubstring("required"))
+						Expect(err.Error()).To(ContainSubstring(config.LabelWorkloadLivenessProbeHTTPPath))
+					})
+
+					It("fails when http probe has missing port", func() {
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeHTTPPath] = "/status"
+
+						err := serviceConfig.validate()
+						Expect(err).To(HaveOccurred())
+						Expect(err.Error()).To(ContainSubstring("required"))
+						Expect(err.Error()).To(ContainSubstring(config.LabelWorkloadLivenessProbeHTTPPort))
+					})
+
+					It("succeeds when all values are provided", func() {
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeHTTPPort] = "/status"
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeHTTPPath] = "8080"
+
+						err := serviceConfig.validate()
+						Expect(err).To(Succeed())
+					})
+				})
+			})
+
+			Context("tcp probe validation", func() {
+				Context("readiness", func() {
+					BeforeEach(func() {
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeType] = kubernetes.ProbeTypeNone.String()
+						serviceConfig.Labels[config.LabelWorkloadReadinessProbeType] = kubernetes.ProbeTypeTCP.String()
+					})
+
+					It("fails when tcp probe has missing port", func() {
+						err := serviceConfig.validate()
+						Expect(err).To(HaveOccurred())
+						Expect(err.Error()).To(ContainSubstring("required"))
+						Expect(err.Error()).To(ContainSubstring(config.LabelWorkloadReadinessProbeTCPPort))
+					})
+
+					It("succeeds when port is provided", func() {
+						serviceConfig.Labels[config.LabelWorkloadReadinessProbeTCPPort] = "8080"
+
+						err := serviceConfig.validate()
+						Expect(err).To(Succeed())
+					})
+				})
+				Context("liveness", func() {
+					BeforeEach(func() {
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeType] = kubernetes.ProbeTypeTCP.String()
+					})
+
+					It("fails when tcp probe has missing port", func() {
+						err := serviceConfig.validate()
+						Expect(err).To(HaveOccurred())
+						Expect(err.Error()).To(ContainSubstring("required"))
+						Expect(err.Error()).To(ContainSubstring(config.LabelWorkloadLivenessProbeTCPPort))
+					})
+
+					It("succeeds when port is provided", func() {
+						serviceConfig.Labels[config.LabelWorkloadLivenessProbeTCPPort] = "8080"
+
+						err := serviceConfig.validate()
+						Expect(err).To(Succeed())
+					})
+				})
+			})
+
+			Context("command probe validation", func() {
+				Context("rediness probess", func() {
```

Comment:
`rediness probess` -> `readiness probe`?

Or just `readiness`? maybe the context below can also be just `liveness`?